kvs/src/route/address_handler.cpp

```diff
@@ -42,24 +42,16 @@ void address_handler(
     respond = true;
   } else {  // if there are servers, attempt to return the correct threads
     for (const Key& key : addr_request.keys()) {
-      unsigned tier_id = 1;
-      ServerThreadSet threads = {};
+      ServerThreadSet threads = kHashRingUtil->get_responsible_threads(
+          rt.get_replication_factor_connect_addr(), key, false,
+          global_hash_ring_map, local_hash_ring_map, placement, pushers,
+          kAllTierIds, succeed, seed, rt.get_tid(), "");
```

Comment:
Why are we passing in an empty string?

Reply:
Should have been the private IP, but the routing thread doesn't have one exposed.
The private IP is only required if the issuing thread comes from a shared memory tier, so in this case this field is not used. The previous argument `rt.get_tid()` is also not necessary for non-shared memory tier callers